<launch>
  <!-- <param name="alpha" type="double" value="0.1"/>
  <param name="wObstacle" type="double" value="0.2"/>
  <param name="wCurvature" type="double" value="0.2"/>
  <param name="wSmoothness" type="double" value="0.2"/>
  <param name="max_iterations" type="int" value="10"/>
  <param name="vorObsDMax" type="double" value="0.6"/>
  <param name="obsDMax" type="double" value="0.8"/>
  <param name="kappaMax" type="double" value="0.5"/> -->
  <param name="globalmap_frame_id" type="string" value="global_map"/>
  <param name="Bspline_degree" type="int" value="8"/>
  <param name="Bspline_numPoints" type="int" value="100"/>
  <!-- Elevation mapping node -->
  <node pkg="path_planning" type="path_smoother" name="path_smoother" output="screen" />

  <!-- Launch visualizations for the resulting elevation map -->
  <!-- <include file="$(find elevation_mapping_ours)/launch/visualization.launch" /> -->

  <!-- Launch RViz with the demo configuration -->
  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find path_planning)/rviz/path_smooth.rviz" /> -->

</launch>
